CODEOWNERS

```diff
@@ -1,16 +1,21 @@
+# changes 4-12-19 CK
 # Aaron should see all code changes (things not in /pages)
-# Josh should see ALL PRs
-* @niartenyaw @jpe442
+# change to only notify Aaron here
+* @niartenyaw
 
 # Josh take over all content (remove Aaron)
-/pages/ @jpe442
+# change to not notify
+# /pages/ @jpe442
 
 # Sarah for services
-/pages/services/ @SarahStegall @jpe442
+# change to not notify
+# /pages/services/ @SarahStegall @jpe442
```

Comment:
i would just delete this section

Reply:
whichever, I can see us maybe needing to ping /services in the future, but we can just delete it